packages/x-components/src/x-modules/related-tags/components/related-tags.vue

```diff
@@ -15,16 +15,22 @@
       <!--
         @slot Custom content that replaces the RelatedTag component.
         @binding {RelatedTag} relatedTag - Related tag data.
+        @binding {boolean} highlightCurated - True if the curated RTs should be displayed.
        -->
-      <slot name="related-tag" :relatedTag="relatedTag">
-        <RelatedTag :relatedTag="relatedTag">
-          <template #default="{ relatedTag, isSelected }">
+      <slot name="related-tag" v-bind="{ relatedTag, highlightCurated }">
+        <RelatedTag :highlightCurated="highlightCurated" :relatedTag="relatedTag">
+          <template #default="{ relatedTag, isSelected, shouldHighlightCurated }">
+            <!-- eslint-disable max-len -->
             <!--
               @slot Custom content that replaces the RelatedTag default content.
               @binding {RelatedTag} relatedTag - Related tag data.
               @binding {boolean} isSelected - Related tag status.
+              @binding {boolean} highlightCurated - True if the curated RTs should be displayed.
```

Comment:
Here it should be `shouldHighlightCurated`, it refers to the computed property of the `RelatedTag` component.

Reply:
My fault.